corehq/apps/es/es_query.py

```diff
@@ -210,7 +210,16 @@ def scroll(self):
         Run the query against the scroll api. Returns an iterator yielding each
         document that matches the query.
         """
-        result = scroll_query(self.index, self.raw_query, for_export=self.for_export)
+        # FIXME: we should crash if someone tries this:
```

Comment:
Leftover `FIXME` - did you intend to leave this in?

Reply:
(I do agree we should raise, though)

Reply:
Agree that raising seems right if aggregation queries will not work (and if that's not how we use scroll anyway), but even more strongly agree that either way we should remove the commented out code (either by deletion or uncommenting) and `FIXME` comment

Reply:
I added this intentionally and also didn't actually implement because I didn't want to expand the scope of _this_ PR. I agree this is a good change to make, but adding it _now_ would negatively affect the safety of this PR. I'd be happy to follow up with a separate PR that makes this change, but I don't want to bog down _this_ (a rather safe bugfix) PR with it.